src/mgr/DaemonServer.cc

```diff
@@ -787,8 +787,8 @@ bool DaemonServer::_handle_command(
   string prefix;
   cmd_getval(cct, cmdctx->cmdmap, "prefix", prefix);
 
-  dout(4) << "decoded " << cmdctx->cmdmap.size() << dendl;
-  dout(4) << "prefix=" << prefix << dendl;
+  dout(20) << "decoded " << cmdctx->cmdmap.size() << dendl;
+  dout(20) << "prefix=" << prefix << dendl;
```

Comment:
can we keep this one? it's a direct proof that the command message hits `DaemonServer`.

Reply:
Ah ok, so the audit log is no proof that the message was handled?

Reply:
i was afraid that the audit log could be disabled . but it turns out that it cannot. unless  debug-monc is less than 0.

see 

https://github.com/ceph/ceph/blob/d5ed1eaa1ba48c0597903be66a526039e66eac48/src/common/LogClient.cc#L206-L210

so, yeah, you are right. it can serve as a proof.